CHARTER.md

```diff
@@ -102,6 +102,17 @@ Governing Board members receive no remuneration by the MapLibre Organization for
 ### Replacement of Governing Board Members
 In the event of a Governing Board member resigning from the Board during a term, the nominees from the last election with the largest number of votes shall be contacted with a proposal to serve in the Government Board until the next election is called.
 
+### Snap Election
+
+A snap election of the Governing Board, i.e. an election which takes place earlier than the usual one year period, can be called if 50 percent of Voting Members who participated in the last election sign a no-confidence motion.
+
+If a no-confidence motion passes, the current Governing Board shall not take any major decisions until the election. In that time the Governing Board shall:
```

Comment:
Is there a definition for "major decisions"?
Otherwise this looks good, thanks!